src/App.tsx

```diff
@@ -10,32 +12,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```

Comment:
If you saved the whole tab instead of just id, then you wouldn't have to use .find() at all

Reply:
good point from @lefteris-L